server/src/main/java/io/spine/server/BoundedContextBuilder.java

```diff
@@ -90,7 +89,7 @@
 
     /**
      * Event dispatchers to be registered with the context {@link EventBus} and/or
-     * {@link IntegrationBroker} after the Bounded Context creation.
+     * {@link io.spine.server.integration.IntegrationBroker} after the Bounded Context creation.
```

Comment:
Can we have a short name of the class following the fully-qualified name in the link, please?